jitpack.yml

```diff
@@ -0,0 +1,2 @@
+jdk:
+  - openjdk11
```

Comment:
Is this needed?

Reply:
No its not, it was also for publishing. Can you remove it or should I do it?

Reply:
Please remove it and I'll try to release everything today

Reply:
Thanks, I removed it now.